iotdb-core/confignode/src/main/java/org/apache/iotdb/confignode/procedure/ProcedureExecutor.java

```diff
@@ -347,6 +347,10 @@ private void executeProcedure(Procedure<Env> proc) {
           switch (executeRootStackRollback(rootProcId, rootProcStack)) {
             case LOCK_ACQUIRED:
               break;
+            case LOCK_EVENT_WAIT:
```

Comment:
Please also check the logic below:

if (!proc.wasExecuted()) {
  ...

Reply:
changed